attributes

```diff
@@ -0,0 +1,95 @@
+[nextcloudUser]
+Type=ExtendedOption
+ObjectClass=nextcloudUser
+Module=users/user
+Default=1
+Editable=1
+Description=Nextcloud
```

Comment:
isn't it a bit short / undeclarative? The original is 'Access to Nextcloud'. Maybe I am wrong and that's ok to present the brief one like that.